numba/targets/codegen.py

```diff
@@ -227,10 +229,13 @@ class BaseCPUCodegen(object):
     def __init__(self, module_name):
         self._libraries = set()
         self._data_layout = None
-        self._llvm_module = ll.parse_assembly(
-            str(self._create_empty_module(module_name)))
+        self._llvm_module = self._materialize_module(
+            self._create_empty_module(module_name))
         self._init(self._llvm_module)
 
+    def _materialize_module(self, ir_module):
```

Comment:
There are also `_create_empty_module` in https://github.com/numba/numba/pull/948/files#diff-ec37b531d0f5e6e4ff2bb91a63bd651dR35